.snyk

```diff
@@ -16,4 +16,5 @@ ignore:
   'SNYK-JAVA-ORGGLASSFISHJERSEYCORE-14049172': *spark-core-exclusions
   'SNYK-JAVA-ORGLZ4-14151788': *spark-core-exclusions
   'SNYK-JAVA-ORGLZ4-14219384': *spark-core-exclusions
+  'SNYK-JAVA-IOAIRLIFT-14412703': *spark-core-exclusions
```

Comment:
I will remove this, just temporarily unblock, if we want this in, should be separated isolated PR

Reply:
Ok did not bite